management_api.go

```diff
@@ -411,10 +411,38 @@ func (e *Enforcer) UpdateNamedGroupingPolicies(ptype string, oldRules [][]string
 
 // RemoveFilteredNamedGroupingPolicy removes a role inheritance rule from the current named policy, field filters can be specified.
 func (e *Enforcer) RemoveFilteredNamedGroupingPolicy(ptype string, fieldIndex int, fieldValues ...string) (bool, error) {
-	return e.removeFilteredPolicy("g", ptype, fieldIndex, fieldValues...)
+	return e.removeFilteredPolicy("g", ptype, fieldIndex, fieldValues)
 }
 
 // AddFunction adds a customized function.
 func (e *Enforcer) AddFunction(name string, function govaluate.ExpressionFunction) {
 	e.fm.AddFunction(name, function)
 }
+
+func (e *Enforcer) SelfAddPolicy(sec string, ptype string, rule []string) (bool, error) {
+	return e.addPolicy(sec, ptype, rule)
```

Comment:
This is what should be called ``addPolicyWithoutNotify``

Reply:
Uhh? My test fails to check it……

Reply:
Fixed, as well as the test case.